src/BloomBrowserUI/templates/bloom-foundation-mixins.pug

```diff
@@ -197,6 +197,23 @@ mixin editable(language)
 	//- .bloom-content1.bloom-editable(lang=language, contenteditable="true")&attributes(attributes)
 		block
 
+//- Putting "+clickableLink('href', boolean) here" should result in a clickable link with correct spacing.
+//- Use the boolean argument 'periodAfter' to insert a period directly behind the link.
+//- Putting "+clickableLink('href')#someId here" should give "<a id=someId ..."
+mixin clickableLink(href, periodAfter)
```

Comment:
The only reason I made it clickableLink() instead of just link() was because there is already a link() element in the stylesheet mixin (to produce CSS <link> elements in the <head> HTML). But I suppose we could still use link() for a mixin, since that would always show up as +link(). The periodAfter was an attempt to keep from having a space between the hyperlink and the period.